shrine/lib/category_menu_page.dart

```diff
@@ -21,45 +21,51 @@ import 'model/product.dart';
 
 class CategoryMenuPage extends StatelessWidget {
   final List<Category> _categories = Category.values;
+  final VoidCallback onCategoryTap;
 
   const CategoryMenuPage({
     Key key,
-  });
+    this.onCategoryTap,
+  }) : super(key: key);
 
   Widget _buildCategory(Category category, BuildContext context) {
     final categoryString =
-        category.toString().replaceAll('Category.', '').toUpperCase();
+    category.toString().replaceAll('Category.', '').toUpperCase();
     final ThemeData theme = Theme.of(context);
     return ScopedModelDescendant<AppStateModel>(
       builder: (context, child, model) => GestureDetector(
-            onTap: () => model.setCategory(category),
-            child: model.selectedCategory == category
-                ? Column(
-                    children: <Widget>[
-                      SizedBox(height: 16.0),
-                      Text(
-                        categoryString,
-                        style: theme.textTheme.body2,
-                        textAlign: TextAlign.center,
-                      ),
-                      SizedBox(height: 14.0),
-                      Container(
-                        width: 70.0,
-                        height: 2.0,
-                        color: kShrinePink400,
-                      ),
-                    ],
-                  )
-                : Padding(
-                    padding: EdgeInsets.symmetric(vertical: 16.0),
-                    child: Text(
-                      categoryString,
-                      style: theme.textTheme.body2
-                          .copyWith(color: kShrineBrown900.withAlpha(153)),
-                      textAlign: TextAlign.center,
-                    ),
-                  ),
+        onTap: () {
+          model.setCategory(category);
+          if (onCategoryTap != null)
+            onCategoryTap();
+        },
+        child: model.selectedCategory == category
+            ? Column(
+          children: <Widget>[
```

Comment:
Did dart format set the indents this way (lines 44-58, 60-66)?

Reply:
No but happy to run it.